doc/classes/String.xml

```diff
@@ -41,6 +41,13 @@
 		</constructor>
 	</constructors>
 	<methods>
+		<method name="bbcode_escape" qualifiers="const">
+			<return type="String" />
+			<description>
+				Returns the string with BBCode tags escaped to [code][lb][/code] and [code][rb][/code], which makes them ineffective when used in [RichTextLabel] with [member RichTextLabel.bbcode_enabled] set to [code]true[/code]. This is useful for handling user input to prevent BBCode injection. See also [method strip_bbcode].
```

Comment:
```suggestion
				Returns the string with BBCode tags escaped with [code][lb][/code], which makes them ineffective when used in [RichTextLabel] with [member RichTextLabel.bbcode_enabled] set to [code]true[/code]. This is useful for handling user input to prevent BBCode injection. See also [method strip_bbcode].
```

The same below.